src/db/migrations/19-add-approved-entities.ts

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
this migration to add the approved field is coming after the migrations to add all the platforms? so the approved field won't really exist yet in the past migrations?

Reply:
ah yeah, will include it int the bootstrap file too